index.html

```diff
@@ -64,98 +92,122 @@ <h2 class="Title-post-1">Multi-Post Stories</h2>
-                                        <li class="tag "><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
-                                    </ul>
+                                <div class="card-7 card card-sec-6 card-fill">
+                                    
+                                        <h2 class="Title-post mob">Profesional Art Printing Data</h2>
+                                        <h2 class="Title-post desk">Website Protfolio </h2>
+                                        <p class="Supporting-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                                    
+                                    <div class="tags">
+                                        <div>
+                                            <ul  class="tag-link">
+                                                <li class="tag "><a href="https://www.html.org">html</a></li>
+                                                <li class="tag "><a href="https://getbootstrap.com/">bootstrap</a></li>
+                                                <li class="tag "><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+                                            </ul>
+                                        </div>
+                                        <span class="see-project"><a href="">See project</a></span>
+                                    </div>
+                                    
                                 </div>
-                                <span class="see-project"><a href="">See project</a></span>
-                            </div>
-                            
-                        </div>
-                    </div> 
+                            </div> 
                 </section>
                 <section>
                     <div class="About">
-                        <h2 class="about-me">About me</h2>
-                        <span><p class="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></span>
-                        <span class="get-resume"><a href="">Get My Resume</a></span>
-                        <div class="image-2">
-
+                        
+                            <h2 class="about-me">About me</h2>
+                            <span><p class="description description-abt">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></span>
+                            <span class="get-resume"><a href="">Get My Resume</a></span>
+                            <div class="image-2">
+                        
```

Comment:
#### In the next section from the highlighted one:

The bullet points are different from the ones mentioned in the FIGMA design. Also look at the `Skill` word. It is pushed forward
Try to make a design as close to FIGMA design as possible.

| Your Design  | FIGMA Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/213814311-419fabf0-fc91-4a02-ac0e-725cc42064c3.png) | ![image](https://user-images.githubusercontent.com/61361037/213814141-6f509108-01c0-49f3-b785-cc1ff119098c.png)  |